docs/index.asciidoc

```diff
@@ -31,6 +31,20 @@ Files ending in `.gz` are handled as gzip'ed files.
 
 Files that are archived to AWS Glacier will be skipped.
 
+==== Event Metadata and the Elastic Common Schema (ECS)
+This plugin adds cloudfront metadata to event.
+By default, the value is stored in the root level.
+When ECS is enabled, it is stored in the `@metadata`.
```

Comment:
🤔 we place it on `@metadata`, but a sharp edge could be users expecting it to also be persisted, and most output plugins avoid persisting `@metadata`. I think if we indicate a clue as to _why_ we place it on `@metadata` here, users who have read the docs will be a little less surprised.

```suggestion
When ECS is enabled, it is stored in the `@metadata` where it can be used by other plugins in your pipeline.
```